ceph_deploy/install.py

```diff
@@ -28,8 +28,48 @@ def sanitize_args(args):
     return args
 
 
+def detect_components(args, distro):
+    """
+    Since the package split, now there are various different ceph components to
+    install like:
+
+    * ceph
+    * ceph-mon
+    * ceph-osd
+    * ceph-mds
+
+    This helper function should parse the args that may contain specifics about
+    these flags and return the default if none are passed in (which is, install
+    everything)
+    """
+    flags = {
+        'install_osd': 'ceph-osd',
+        'install_rgw': 'ceph-radosgw',
+        'install_mds': 'ceph-mds',
+        'install_mon': 'ceph-mon',
+        'install_ceph': 'ceph',
```

Comment:
@ktdreyer I could use your help here...  Do we need "ceph" at all here?  Ceph is a dependency of ceph-osd, ceph-radosgw, ceph-mon, and ceph-mds, right?  In other words, once we do a package split, we never need to in install "ceph".  It's always ceph-*, which will pull in ceph.